microsoft-edge/webview2/concepts/data_privacy.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Data & Privacy in WebView2
+description: Overview of WebView2 Data Collection and Practices. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023
+---
+# Data & Privacy in WebView2 

```

Comment:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/ampersand -- "Don't use _&_ in place of _and_ in text or headings unless you're referring to the use of the symbol in UI."

If change title, change list of usual places:
* title: YAML metadata line 2.
* h1 line 11
* toc.yml
* Other pages that link to here.